File: proyecto-jueves/src/app/model/employee.ts
export class Employee {

    
    constructor(private _id:number,private _name:string,private _phoneNumber:string,private _officeNumber:string,private _position:string,private _manager:string) {}

    public get id():number {
        return this._id
    }
   public set id(id:number) {
        this._id=id
    }

    public get name():string {
        return this._name
    }
    public set name(name:string) {
        this._name=name
    }

    public get phoneNumber():string {
        return this._phoneNumber
    }
    public set phoneNumber(phoneNumber:string) {
        this._phoneNumber=phoneNumber
    }

    public get officeNumber():string {
        return this._officeNumber
    }
    public set officeNumber(officeNumber:string) {
        this._officeNumber=officeNumber
    }

    public get position():string {
        return this._position
    }
    public set position(position:string) {
        this._position=position
    }

    public get manager():string {
        return this._manager
    }
    public set manager(manager:string) {
        this._manager=manager
    }
}

File: jose/src/main/java/com/ironhack/jose/models/Employee.java
package com.ironhack.jose.models;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Employee {

    public Employee(String name, String phoneNumber, String officeNumber, String position, String manager) {
        this.name = name;
        this.phoneNumber = phoneNumber;
        this.officeNumber = officeNumber;
        this.position = position;
        this.manager = manager;
    }

    public Employee() {

    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    private String name;
    private String phoneNumber;
    private String officeNumber;
    private String position;
    private String manager;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getOfficeNumber() {
        return officeNumber;
    }

    public void setOfficeNumber(String officeNumber) {
        this.officeNumber = officeNumber;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getManager() {
        return manager;
    }

    public void setManager(String manager) {
        this.manager = manager;
    }
}
